include/gz/gui/Application.hh

```diff
@@ -98,6 +98,8 @@ namespace gz::gui
       /// \sa InitializeDialogs
       public: bool LoadConfig(const std::string &_path);
 
+      public: bool LoadWindowConfig(const tinyxml2::XMLElement &_window);
```

Comment:
add doxygen comments for the new functions?

Reply:
Done in https://github.com/gazebosim/gz-gui/pull/631/commits/c2fa7925ee8c49e846957018dba9adc6b4a3202e